utils/ixdtf/src/parsers/records.rs

```diff
@@ -141,13 +214,11 @@ impl UtcOffsetRecordOrZ {
     pub fn resolve_rfc_9557(self) -> UtcOffsetRecord {
         match self {
             UtcOffsetRecordOrZ::Offset(o) => o,
-            UtcOffsetRecordOrZ::Z => UtcOffsetRecord {
+            UtcOffsetRecordOrZ::Z => UtcOffsetRecord::MinutePrecision(MinutePrecisionOffset {
```

Comment:
why was this `second:0, fraction: None` before, i.e. why is this not `FullPrecisionOffset` now?

Reply:
RFC 9557 only supports the updated definition of `-00:00`